EasyPost.Tests/ClientTest.cs

```diff
@@ -21,19 +19,19 @@
 
 //        [TestMethod]
 //        public void TestApiBase() {
-//            Client client = new Client(new ClientConfiguration("apiKey", "https://foobar.com"));
+//            var client = new Client(new ClientConfiguration("apiKey", "https://foobar.com"));
```

Comment:
Were we changing all the `var`s back to the type of variable it was? I think vars appear in a couple files.

Reply:
Yes. Since this is commented out, the linting tools didn't act on it. I'll fix it manually.

Reply:
Resolved in 9fe4dc7897fab6d20927ee3ad9c6e9e75aa64a07

Reply:
Resolved in 9fe4dc7897fab6d20927ee3ad9c6e9e75aa64a07